### PR TITLE
add first R packages

### PR DESCRIPTION
**Pull request by bgruening**
This will also serve to test travis.

### CODE REVIEW

**Comment by nturaga** [comment]
@bgruening I'm guessing bioarchive is not giving you the packages you wanted?

**Comment by bgruening** [comment]
@nitesh1989 ? What do you mean?
I need some tarballs to build old recipes. With these tools everything is ok. Just building conda packages out of it.

**Comment by bgruening** [comment]
@galaxyproject/iuc please have a look at this PR.

This is not a nice conda recipe, but it is more or less automatically converted from tool_dependencies.xml files. it will test the package and upload it to anaconda. The result can be seen here:

https://anaconda.org/iuc/bioconductor-ballgown

File: recipes/bioconductor-ballgown/1.0.3/build.sh [hunk 1]
```diff
@@ -0,0 +1,393 @@
...
+# For whatever reason, it can't link to gsl correctly without this on OS X.
+export DYLD_FALLBACK_LIBRARY_PATH=$PREFIX/lib:$(HOME)/lib:/usr/local/lib:/lib:/usr/lib
+
+
+wget --no-check-certificate https://depot.galaxyproject.org/package/noarch/BiocGenerics_0.12.1.tar.gz
+sha256sum BiocGenerics_0.12.1.tar.gz | awk '{ if ($1 == "d451f2c72c46a7b5fa6dd252a207ba72f0a5a86d1bfc95408935f079efa85f77") exit 0; else exit 1}'
+tar xfz BiocGenerics_0.12.1.tar.gz
+cd BiocGenerics
+mv DESCRIPTION DESCRIPTION.old
+grep -v '^Priority: ' DESCRIPTION.old > DESCRIPTION
```
Comment (nsoranzo) [inline recipes/bioconductor-ballgown/1.0.3/build.sh:12]: Why not use

```
sed -i.old -e '/^Priority: /d' DESCRIPTION
```

instead of mv && grep -v?

**Pull request merged**